cmake/stm32/h5.cmake

```diff
@@ -0,0 +1,23 @@
+set(STM32_H5_TYPES 
+    H503xx H562xx H563xx H573xx
+)
+
+set(STM32_H5_TYPE_MATCH 
+    "H503.." "H562.." "H563.." "H573.."
+)
+set(STM32_H5_RAM_SIZES 
+    32K 640K 640K 640K
+)
+set(STM32_H5_CCRAM_SIZES 
+    0K  0K  0K  0K
+)
+
+stm32_util_create_family_targets(H5)
+
+target_compile_options(STM32::H5 INTERFACE 
+    -mcpu=cortex-m7 -mfloat-abi=hard -mfpu=fpv5-d16 -mthumb
```

Comment:
Just a notice from someone following the project:  
As the H5 is a Cortex-M55 i did expect `-mcpu=cortex-m33` here and below.

Reply:
Correct. You mean M33 not M55 I suppose ?

Reply:
Yes, did correct it in the initial comment

Reply:
Yes of course